.ci/integrationTestECK.groovy

```diff
@@ -57,14 +57,17 @@ pipeline {
         agent { label 'linux && immutable' }
         axes {
           axis {
-              name 'ELASTIC_STACK_VERSION'
-              values '7.13.0-SNAPSHOT', '7.12.1-SNAPSHOT', '7.12.0'
+            name 'STACK_VERSION'
+            // The below line is part of the bump release automation
+            // if you change anything please modifies the file
+            // .ci/bump-stack-release-version.sh
+            values '7.x', '7.12.0'
```

Comment:
ditto.

Let's test with the latest 7 snapshot for simplicity

Reply:
Or do we need to support 8.0-SNAPSHOT too?

Reply:
last time I tested 8.0.0-SNAPSHOT did not work with ECK, but probably it do now

Reply:
I'll try it once this is merged in another follow up then